pages/new_exercise.php

```diff
@@ -0,0 +1,126 @@
+include_once($_SERVER['DOCUMENT_ROOT'] . '/html/head.php');
+include_once($_SERVER['DOCUMENT_ROOT'] . '/html/header.php');
+?>
+
+<head>
+    <link rel="stylesheet" href="../css/new_exercise.css">
+</head>
+
+<div class="container">
+    <div class="header">
+        <h3>Neue Aufgabe erstellen</h3>
+    </div>
+
+    <hr>
+
+    <form action="<?= $action_path ?>" method="post">
+        <input type="hidden" name="user_id" value="<?= $user->get_id() ?>">
+        <div class="row">
+            <!-- erste Spalte -->
+            <div class="col s12 m6">
+                <div class="input-field ">
+                    <input type="text" name="title" id="title">
+                    <label for="title">Aufgabentitel</label>
+                </div>
+                <div class="input-field ">
+                    <textarea type="text" name="description" id="description" class="materialize-textarea"></textarea>
+                    <label for="description">Aufgabenstellung</label>
+                </div>
+                <div class="input-field ">
+                    <input type="text" name="solution" id="solution">
+                    <label for="solution">Lösung</label>
+                </div>
+
+                <button type="submit" class="waves-effect waves-light btn">Erstellen</button>
+            </div>
+
+            <!-- zweite Spalte -->
+            <div class="col s12 m6">
+                <div class="input-field">
+                    <select name="category">
+                        <option value="" selected disabled>Bitte etwas auswählen</option>
+                        <?php foreach ($categories as $category) { ?>
+                            <option value="<?= $category->get_id() ?>"><?= $category->get_description() ?></option>
+                        <?php } ?>
+                    </select>
+                    <label for="category">Kategorie</label>
+                </div>
+                <div class="input-field">
+                    <select name="subcategory">
+                        <option value="" selected disabled>Bitte etwas auswählen</option>
+                        <?php foreach ($subcategories as $subcategory) { ?>
+                            <option value="<?= $subcategory->get_id() ?>"><?= $subcategory->get_description() ?></option>
+                        <?php } ?>
+                    </select>
+                    <label for="category">Unterkategorie</label>
+                </div>
+                <div class="input-field">
+                    <select name="difficulty">
+                        <option value="" selected disabled>Bitte etwas auswählen</option>
+                        <?php foreach ($difficulties as $difficulty) { ?>
+                            <option value="<?= $difficulty->get_id() ?>"><?= $difficulty->get_description() ?></option>
+                        <?php } ?>
+                    </select>
+                    <label for="category">Schwierigkeit</label>
+                </div>
```

Comment:
Könntest DU noch Beispiel-Kategorien/Unterkategorien/Schwierigkeiten in das php Skript einfügen, damit man das besser testen kann und es besser aussieht? 🙈

Reply:
Du musst einfach nur mehr Daten bei dir in der Datenbank einfügen. Dann werden auch mehr angezeigt.